lib/note-content-editor.jsx

```diff
@@ -1,11 +1,68 @@
 import React, { PropTypes } from 'react';
-import { Editor, EditorState, ContentState } from 'draft-js';
+import {
+	ContentState,
+	Editor,
+	EditorState,
+	Modifier,
+} from 'draft-js';
 import { invoke, noop } from 'lodash';
 
 function plainTextContent( editorState ) {
 	return editorState.getCurrentContent().getPlainText( '\n' )
 }
 
+// contains methods that mirror the methods on Modifier with 2 differences:
+// 1. operate on EditorState instances
+// 2. move selection to the natural place following the modification
```

Comment:
I like this comment, but I find it a big lacking in content. For example, I would assume that the normal `insertText` function in `draft-js` would make the claim that it moves the selection to the natural place following a modification, but we have a different definition of `natural`

(sorry for duplicating the same comment here as in the other PR), but I would be strongly in favor of not hiding these methods inside the `EditorStateModifier` object. Even simply extracting them without changing their names would be better, though it's possible that there could be more fitting names.

We can then turn the comments into proper JSDoc DocBlock comments explaining the behaviors, such as "Moves the selection to the end of the insertion"